docs/Analyzer Configuration.md

```diff
@@ -474,3 +474,30 @@ Examples:
 | --- | --- |
 |`dotnet_code_quality.CA1010.additional_required_generic_interfaces = ISomething->System.Collections.Generic.IEnumerable`\``1` | All types implementing 'ISomething' regardless of its namespace are expected to also implement 'System.Collections.Generic.IEnumerable`1'. |
 |`dotnet_code_quality.CA1010.additional_required_generic_interfaces = T:System.Collections.IDictionary->T:System.Collections.Generic.IDictionary`\``2` | All types implementing 'System.Collections.Generic.IDictionary' are expected to also implement 'System.Collections.Generic.IDictionary`2'. |
+
+### Inheritance excluded type or namespace names
+Option Name: `additional_inheritance_excluded_symbol_names`
+
+Configurable Rules: [CA1501](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1501)
+
+Option Values: Names of types or namespaces (separated by '|'), such that the type or type's namespace doesn't count in the inheritance hierarchy tree.
+Allowed symbol name formats:
+  1. Type or namespace name (includes all types with the name, regardless of the containing type or namespace and all types whose namespace contains the name)
+  2. Type or namespace name ending with a wildcard symbol (includes all types whose name starts with the given name, regardless of the containing type or namespace and all types whose namespace contains the name)
+  3. Fully qualified type or namespace name with an optional "T:" prefix for type or "N:" prefix for namespace (includes all types with the exact type match or the exact containing namespace match)
```

Comment:
Why was the 3rd bullet point changed to exclude documentation ID format and the help link? Only the wildcard based option value does not abide by that format. Non-wildcard based options still must be in that format and we should document these similar to other options.

Reply:
Done